monai/utils/enums.py

```diff
@@ -259,6 +260,13 @@ class CommonKeys:
     LOSS = "loss"
 
 
+class DictPostFixes:
+    """Post-fixes."""
+
+    META = "meta_dict"
+    ORIG_META = "orig_meta_dict"
```

Comment:
shall we add this refactoring:
```py
class DictPostFixes:
    """Post-fixes."""

    META = "meta_dict"
    ORIG_META = "orig_meta_dict"

    @staticmethod
    def meta_key(key):
        return f"{key}_{DictPostFixes.META}"
```
usage:
```py
DictPostFixes.meta_key("image")  # "image_meata_dict"
```

Reply:
Great idea, I'll update now.